### ISSUE TITLE
v12 regression: adding accessibility fails when using npm module

### ISSUE THREAD

**Issue by anajavi** [highcharts/highcharts#22251]
#### Expected behaviour
```js
import * as Highcharts from 'highcharts';
import addAccessibility from 'highcharts/modules/accessibility';

addAccessibility(Highcharts);
```
Works as it works on v11.

#### Actual behaviour

On V12 it crashes with `(0 , accessibility_1.default) is not a function`

Same thing with other modules.

#### Live demo with steps to reproduce
[demo](https://stackblitz.com/edit/react-starter-typescript-2aye8x?file=package.json,App.tsx)

on v12 the demo crashes with `(0 , accessibility_1.default) is not a function`

downgrading highcharts to V11 the demo does not crash.

#### Product version
Highcharts v12.0.1

#### Affected browser(s)
- Chrome 131
- Safari 18.1.1

**Comment by TorsteinHonsi** [comment]
Thanks for reporting, we will look into this!

**Comment by bre1470** [comment]
Thank you for the report. In version 12 you do not have to call module factories. You can remove the call. We will update our documentation.

**Comment by cintaccs** [comment]
I have a similar module problem - this worked prior to version 12:
```
import highstock from 'highcharts/highstock';
import more from 'highcharts/highcharts-more';
import thirdDim from 'highcharts/highcharts-3d';

import funnel from 'highcharts/modules/funnel';
funnel(highstock);
more(highstock);
thirdDim(highstock);

import cylinder from 'highcharts/modules/cylinder';
cylinder(highstock);

import map from 'highcharts/modules/map';
map(highstock);
```

I get various errors like ... funnel is not a function.
if I remove the funnel(Highstock) part - I get 

````
Error: Highcharts error #17: www.highcharts.com/errors/17/?missingModuleFor=packedbubble
 - missingModuleFor: packedbubble
`````
etc.

**Comment by anajavi** [comment]
> Thank you for the report. In version 12 you do not have to call module factories. You can remove the call. We will update our documentation.

This works correctly for highcharts:
```js
import 'highcharts/modules/accessibility';
```

However for highmaps and highstock it does not remove warning from console:
```
Highcharts warning: Consider including the "accessibility.js" module to make your chart more usable for people with disabilities. Set the "accessibility.enabled" option to false to remove this warning. See https://www.highcharts.com/docs/accessibility/accessibility-module.
```

**Comment by DiegoCardoso** [comment]
I can confirm that the modules in version 12 work correctly with Highcharts, but they don't currently don't work if you import Highstock instead. It seems as if the global instance being picked by the modules is different from the one used by Highstock.

**Comment by anajavi** [comment]
> I can confirm that the modules in version 12 work correctly with Highcharts, but they don't currently don't work if you import Highstock instead. It seems as if the global instance being picked by the modules is different from the one used by Highstock.

@DiegoCardoso I created a new issue for the highstock problem: #22258, closing this as the change was intentional.

**Comment by httpete-broadcom** [comment]
Hitting the same thing here, using esbuild in a monorepo. Highcharts should be repackaged to provide an option that does not use any globals and allows modern loaders to connect and use the modules.

**Comment by hubertkozik** [comment]
Hi @httpete-broadcom! I think your issue might be connected with https://github.com/highcharts/highcharts/issues/22318 and the other topics mentioned there.
